### ISSUE TITLE
Changes view with a screen reader active does not announce "No changes"

### ISSUE THREAD

**Issue by chrmarti** [microsoft/vscode#309476]
Testing #309100

The Changes view with a screen reader active does not announce "No changes". The Changes view does not appear to take focus, maybe that's why the announcement is missing:

<img width="1646" height="1023" alt="Image" src="https://github.com/user-attachments/assets/4aea5f84-7e68-446c-bedf-009e0b9a28d2" />
